src/sql/sqlops.proposed.d.ts

```diff
@@ -362,8 +362,26 @@ declare module 'sqlops' {
 			 */
 			export function closeDialog(dialog: Dialog): void;
 
+			export interface ModelViewPanel {
+				/**
+				 * Register model view content for the dialog.
+				 * Doesn't do anything if model view is already registered
+				 */
+				registerContent(handler: (view: ModelView) => void): void;
+
+				/**
+				 * Returns the model view content if registered. Returns undefined if model review is not registered
+				 */
+				readonly modelView: ModelView;
+
+				/**
+				 * The handle assigned to the UI
+				 */
+				handle: number;
```

Comment:
I don't think this should be part of the API. There's no reason for an extension author to ever need it is there?